File: AtCoder/past201912-open/d/Main.php
<?php

$n = (int) fgets(STDIN);

$arr = array_fill(1, $n, 0);
for ($i = 1; $i <= $n; $i++) { 
    $arr[(int)fgets(STDIN)]++;
}

$before = -1;
$after = -1;
for ($i = 1; $i <= $n; $i++) {
    if ($arr[$i] == 0) $before = $i;
    if ($arr[$i] == 2) $after = $i;
}
if ($before == -1) print "Correct";
else print $after . " " . $before;

File: AtCoder/past201912-open/d/main.py
n = int(input())
cnt = {x: 0 for x in range(1, n + 1)}

for i in range(n):
    cnt[int(input())] += 1

before = -1
after = -1
for i in range(1, n + 1):
    if cnt[i] == 0:
        before = i
    if cnt[i] == 2:
        after = i

if before == -1:
    print("Correct")
else:
    print(after, before)
